specification/cognitiveservices/data-plane/AzureOpenAI/authoring/preview/2023-04-15-preview/images.json

```diff
@@ -0,0 +1,527 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure OpenAI DALL-E API version 2023-04-15-preview",
+    "description": "Cognitive Services API to support OpenAI DALL-E",
+    "version": "2023-04-15-preview"
+  },
+  "host": "eastus.api.cognitive.microsoft.com",
+  "basePath": "/dalle",
+  "parameters": {
+    "endpoint": {
+      "in": "path",
+      "name": "endpoint",
+      "description": "Supported Cognitive Services endpoints (protocol and hostname, for example: https://aoairesource.openai.azure.com. Replace \"aoairesource\" with your Azure OpenAI account name).",
+      "required": true,
+      "type": "string",
+      "format": "url",
+      "x-ms-skip-url-encoding": true,
+      "x-ms-parameter-location": "client"
+    },
+    "apiVersionQueryParameter": {
+      "in": "query",
+      "name": "api-version",
+      "description": "The requested API version.",
+      "required": true,
+      "type": "string",
+      "x-ms-client-default": "2023-04-15-preview",
+      "x-ms-parameter-location": "client"
+    }
+  },
+  "paths": {
+    "/images/generate": {
```

Comment:
>/ [](http://example.com/codeflow?start=12&length=1)

:, action pattern.

Reply:
As mentioned, this is on purpose as we wanted to stay as close as possible to openAI.